# single positions

floor_pickup: [[1.2912056067720998, -1.604857778519845, -0.021189260004231159, -0.14217745804097515, 2.9290525086918109, 0.47424578989074639, 0.056571318759243816]]

pre_drop: [[1.291767326820394, -1.5804211634448446, -0.022503857040611308, 0.20725920520452767, 2.9281959309366314, 0.5631037482140373, 0.057166136243057331]]

drop: [[1.285391531519128, -1.6823948149997605, -0.022919689278168676, 0.30770523994868881, 2.9273442873757105, 0.13829395205235695, 0.056130105893574367]]

grab_milk: [[-0.054294896454149644, -1.6722009011269954, -2.4761469082680563, -0.32395537616114733, 2.0163673203955277, 1.0301744078651645, -2.1365785809470541]]

pre_milk: [[-0.16713712863801433, -1.7676949458811508, -2.4717864946888861, -0.20466913226826033, 1.7231864350764781, 1.3595725324228223, -1.9123890362877232]]

crash: [[0.43975259544199474, -1.9117397123845157, -2.1465126643531409, 0.1426001442350362, 2.0156458373958284, 1.0296534312652801, -2.1365526166804027]]

